js/application/init.js

```diff
@@ -242,7 +247,12 @@ define([
           document.title = this.config.title;
 
           // Launch the app specifics
-          this.main.start(this.config, dom.byId("viewDiv"), view, webscene, null, null);
+          this.main.start({
+            config: this.config,
+            viewDiv: dom.byId("viewDiv"),
+            view: view,
+            mapOrScene: webscene
```

Comment:
Yeah, but lets just tell them what it is upfront so they don't have to do that. There's really no reason to have it as a combined property that handles different things.